pygfx/materials/_mesh.py

```diff
@@ -506,8 +506,113 @@ def shininess(self, value):
     # TODO: more advanced mproperties, Unified with "MeshStandardMaterial".
 
 
-# todo: MeshToonMaterial(MeshBasicMaterial):
-# A cartoon-style mesh material.
+class MeshToonMaterial(MeshBasicMaterial):
+    """
+    A material implementing toon shading.
+
+    Parameters
+    ----------
+    emissive : Color
+        The emissive (light) color of the mesh. This color is added to the final
+        color and is unaffected by lighting. The alpha channel of this color is
+        ignored.
+
```

Comment:
Let's also include `normal_map`?

Reply:
I've noticed that the constructor for `MeshStandardMaterial `also lacks parameters for `normal_map` (and some other properties aren't listed in the Material constructor's parameters either). It seems like there isn't a standardized approach here.

For advanced materials, there may be many optional properties (which might be added gradually). Should all these optional properties be listed in the constructor method? Perhaps the constructor parameters should only include the most crucial properties of the current material (those that users need to focus on and significantly affect the material's appearance)?

Reply:
Good point. The approach is that all properties are settable in the constructor. So the other missing cases should (at some point) be fixed.

As for the docs, I created #661 earlier. I think we should stop duplicating the docstrings. I added a comment in that issue.

For now, in this PR, maybe just add the most important once, and we can correct it all when we implement #661